src/components/InventoryManager/ListingInputField.js

```diff
@@ -76,7 +83,14 @@ function AutoCompleteInputField({
           autoComplete
           options={options}
           getOptionLabel={getLabel}
-          onChange={onChange}
+          getOptionSelected={(option, value) => (
```

Comment:
this part should make it so that it doesn't crash if you click the x and then outside of the field.
now it shoud reset the field to whatever it was before you erased it